dataland-frontend/tests/e2e/specs/search-company-framework-data/SearchCompaniesForFrameworkDataDropdownFilter.ts

```diff
@@ -82,7 +82,7 @@ describe("As a user, I expect the search functionality on the /companies page to
       )
         .wait("@companies-meta-information")
         .get("div[class='col-12 text-left']")
-        .should("contain.text", "Sorry! Your search didn't return any results.")
+        .should("contain.text", "We're sorry, but your search did not return any results.")
```

Comment:
Can this be moved into a variable. Then we reduce the instances we have to maintain, if something in the text changes.

Reply:
Done

Reply:
Ok